examples/gitlab-merge-request-template/workflow-template-mr.yaml

```diff
@@ -0,0 +1,197 @@
+# SUMMARY:
+#
+# Auto trigger the workflowTemplate once the Gitlab merge request was created or updated.
+#
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: mr-workflow
+  namespace: default
+spec:
+  entrypoint: main
+  arguments:
+    parameters:
+      - name: repo
+        value: git@gitlab.com:linuxsuren/gogit.git
```

Comment:
Can we use other repos instead of personal one?

Reply:
Do you have any suggestions? This is only an example. I don't use Gitlab often. Or change it to a placeholder as well?

Reply:
Yeah perhaps change it to a placeholder then. I don't use GitLab either.

Reply:
All changes made. The self-hosted Gitlab is quite popular in Chinese companies.